### ISSUE TITLE
Can't install latest version of shapley on apple M1 

### ISSUE THREAD

**Issue by shay-te** [shapely/shapely#1298]
Hello, having issue installing shapley using pip
Expecting that Shapley will be install but get the following error 
thank you

Mac OS X 12.1 
Apple M1
request Shapely version 1.8.0

This issue happens using PyCharm with virtual env when the base interpreter is `/usr/bin/python3`

installation working when base interpreter is `/usr/local/bin/python3` with version 3.9.10

```shell
pip install shapely==1.8.0

Collecting shapely==1.8.0
  Using cached Shapely-1.8.0.tar.gz (278 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/shay/Desktop/myapp/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/setup.py'"'"'; __file__='"'"'/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-pip-egg-info-b_9rv_lo
       cwd: /private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/
  Complete output (13 lines):
  Failed `CDLL(/Library/Frameworks/GEOS.framework/Versions/Current/GEOS)`
  Failed `CDLL(/opt/local/lib/libgeos_c.dylib)`
  Failed `CDLL(/usr/local/lib/libgeos_c.dylib)`
  Failed `CDLL(/opt/homebrew/lib/libgeos_c.dylib)`
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/setup.py", line 74, in <module>
      from shapely._buildcfg import geos_version_string, geos_version, \
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/shapely/_buildcfg.py", line 189, in <module>
      lgeos = load_dll('geos_c', fallbacks=alt_paths)
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-o66knqia/shapely_5df5b9f4f9af438da91f46b657d69d40/shapely/_buildcfg.py", line 159, in load_dll
      raise OSError(
  OSError: Could not find library geos_c or load any of its variants ['/Library/Frameworks/GEOS.framework/Versions/Current/GEOS', '/opt/local/lib/libgeos_c.dylib', '/usr/local/lib/libgeos_c.dylib', '/opt/homebrew/lib/libgeos_c.dylib']
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/1c/0c/454c80f71bd5ece52fb06d2905bf956b9122f4be539d5ae5df4b10dd3e14/Shapely-1.8.0.tar.gz#sha256=f5307ee14ba4199f8bbcf6532ca33064661c1433960c432c84f0daa73b47ef9c (from https://pypi.org/simple/shapely/) (requires-python:>=3.6). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement shapely==1.8.0 (from versions: 1.0a7, 1.0b1, 1.0b2, 1.0b3, 1.0b4, 1.0rc1, 1.0rc2, 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.11, 1.0.12, 1.0.13, 1.0.14, 1.0.15, 1.2b1, 1.2b2, 1.2b3, 1.2b4, 1.2b5, 1.2b6, 1.2b7, 1.2rc1, 1.2rc2, 1.2, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.2.8, 1.2.9, 1.2.10, 1.2.12, 1.2.13, 1.2.14, 1.2.15, 1.2.16, 1.2.17, 1.2.18, 1.2.19, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5, 1.5.6, 1.5.7, 1.5.8, 1.5.9, 1.5.10, 1.5.11, 1.5.12, 1.5.13, 1.5.14, 1.5.15, 1.5.16, 1.5.17, 1.6a1, 1.6a2, 1.6b1, 1.6b2, 1.6b3, 1.6b4, 1.6b5, 1.6.0, 1.6.1, 1.6.2, 1.6.2.post1, 1.6.3, 1.6.4, 1.6.4.post1, 1.6.4.post2, 1.7a1, 1.7a2, 1.7a3, 1.7b1, 1.7.0, 1.7.1, 1.8a1, 1.8a2, 1.8a3, 1.8rc1, 1.8rc2, 1.8.0)
ERROR: No matching distribution found for shapely==1.8.0
(venv) MacBook-Pro:core-lib shaytessler$ 

```

after that running 
```shell
brew install geos
```

and then running again have this error 

```shell
pip install shapely==1.8.0
Collecting shapely==1.8.0
  Using cached Shapely-1.8.0.tar.gz (278 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/shay/Desktop/myapp/venv/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/setup.py'"'"'; __file__='"'"'/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-pip-egg-info-a8pr_hvc
       cwd: /private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/
  Complete output (19 lines):
  cannot open shared object from 'geos-config --clibs': '-L/usr/local/Cellar/geos/3.10.2/lib -lgeos_c'
  there were 3 guess(es) for this path:
        /usr/local/Cellar/geos/3.10.2/lib/libgeos_c.dylib
        /usr/local/Cellar/geos/3.10.2/lib/geos_c.dylib
        /usr/local/Cellar/geos/3.10.2/lib/geos_c.framework/geos_c
  Failed `CDLL(/usr/local/lib/libgeos_c.dylib)`
  Failed `CDLL(/Library/Frameworks/GEOS.framework/Versions/Current/GEOS)`
  Failed `CDLL(/opt/local/lib/libgeos_c.dylib)`
  Failed `CDLL(/usr/local/lib/libgeos_c.dylib)`
  Failed `CDLL(/opt/homebrew/lib/libgeos_c.dylib)`
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/setup.py", line 74, in <module>
      from shapely._buildcfg import geos_version_string, geos_version, \
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/shapely/_buildcfg.py", line 189, in <module>
      lgeos = load_dll('geos_c', fallbacks=alt_paths)
    File "/private/var/folders/k9/rsqyflp14j3gjqy2gnd5g78h0000gn/T/pip-install-u_b04d2p/shapely_153556bcaad043d3a871a6e682fa59d2/shapely/_buildcfg.py", line 159, in load_dll
      raise OSError(
  OSError: Could not find library geos_c or load any of its variants ['/Library/Frameworks/GEOS.framework/Versions/Current/GEOS', '/opt/local/lib/libgeos_c.dylib', '/usr/local/lib/libgeos_c.dylib', '/opt/homebrew/lib/libgeos_c.dylib']
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/1c/0c/454c80f71bd5ece52fb06d2905bf956b9122f4be539d5ae5df4b10dd3e14/Shapely-1.8.0.tar.gz#sha256=f5307ee14ba4199f8bbcf6532ca33064661c1433960c432c84f0daa73b47ef9c (from https://pypi.org/simple/shapely/) (requires-python:>=3.6). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement shapely==1.8.0 (from versions: 1.0a7, 1.0b1, 1.0b2, 1.0b3, 1.0b4, 1.0rc1, 1.0rc2, 1.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.0.6, 1.0.7, 1.0.11, 1.0.12, 1.0.13, 1.0.14, 1.0.15, 1.2b1, 1.2b2, 1.2b3, 1.2b4, 1.2b5, 1.2b6, 1.2b7, 1.2rc1, 1.2rc2, 1.2, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.2.6, 1.2.7, 1.2.8, 1.2.9, 1.2.10, 1.2.12, 1.2.13, 1.2.14, 1.2.15, 1.2.16, 1.2.17, 1.2.18, 1.2.19, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1, 1.5.2, 1.5.3, 1.5.4, 1.5.5, 1.5.6, 1.5.7, 1.5.8, 1.5.9, 1.5.10, 1.5.11, 1.5.12, 1.5.13, 1.5.14, 1.5.15, 1.5.16, 1.5.17, 1.6a1, 1.6a2, 1.6b1, 1.6b2, 1.6b3, 1.6b4, 1.6b5, 1.6.0, 1.6.1, 1.6.2, 1.6.2.post1, 1.6.3, 1.6.4, 1.6.4.post1, 1.6.4.post2, 1.7a1, 1.7a2, 1.7a3, 1.7b1, 1.7.0, 1.7.1, 1.8a1, 1.8a2, 1.8a3, 1.8rc1, 1.8rc2, 1.8.0)
ERROR: No matching distribution found for shapely==1.8.0
```

**Comment by christian-steinmeyer** [comment]
Same issue on 
>  Model Name:	MacBook Pro
>  Processor Name:	8-Core Intel Core i9
>  Total Number of Cores:	8
>  System Firmware Version:	1715.60.5.0.0 (iBridge: 19.16.10647.0.0,0)
>  OS Loader Version:	540.60.2~89

in a clean virtual python environment based on python 3.10 and pip@22.0.3

```bash
Collecting Shapely
  Downloading Shapely-1.8.0.tar.gz (278 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 278.3/278.3 KB 455.3 kB/s eta 0:00:00
  Preparing metadata (setup.py) ... error
    error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Failed `CDLL(/Library/Frameworks/GEOS.framework/Versions/Current/GEOS)`
      Failed `CDLL(/opt/local/lib/libgeos_c.dylib)`
      Failed `CDLL(/usr/local/lib/libgeos_c.dylib)`
      Failed `CDLL(/opt/homebrew/lib/libgeos_c.dylib)`
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/private/var/folders/0_/c7t6hlbn6tv7g4sc5n23tcyc0000gn/T/pip-install-p5xttb2l/shapely_47af23ddf5b84b979a1f2c5f321387ca/setup.py", line 74, in <module>
          from shapely._buildcfg import geos_version_string, geos_version, \
        File "/private/var/folders/0_/c7t6hlbn6tv7g4sc5n23tcyc0000gn/T/pip-install-p5xttb2l/shapely_47af23ddf5b84b979a1f2c5f321387ca/shapely/_buildcfg.py", line 189, in <module>
          lgeos = load_dll('geos_c', fallbacks=alt_paths)
        File "/private/var/folders/0_/c7t6hlbn6tv7g4sc5n23tcyc0000gn/T/pip-install-p5xttb2l/shapely_47af23ddf5b84b979a1f2c5f321387ca/shapely/_buildcfg.py", line 159, in load_dll
          raise OSError(
      OSError: Could not find library geos_c or load any of its variants ['/Library/Frameworks/GEOS.framework/Versions/Current/GEOS', '/opt/local/lib/libgeos_c.dylib', '/usr/local/lib/libgeos_c.dylib', '/opt/homebrew/lib/libgeos_c.dylib']
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
```

**Comment by christian-steinmeyer** [comment]
@shay-te is this fixed, if you run `brew install geos` again, followed by `pip install shapely --no-cache-dir`?

**Comment by shay-te** [comment]
> @shay-te is this fixed, if you run `brew install geos` again, followed by `pip install shapely --no-cache-dir`?

Thank you this was tried with no success.

**Comment by zacclery** [comment]
@christian-steinmeyer Worked for me. Thanks.

**Comment by sgillies** [comment]
This is a known, duplicate issue and will be solved by version 1.8.1.

**Comment by cemlyn007** [comment]
You can use conda to install geo as well 😃
